user-guide/Advanced_Functionality/Swarming/EnableSwarming.md

```diff
@@ -0,0 +1,102 @@
+---
+uid: EnableSwarming
+---
+
+# Enabling the Swarming feature
+
+> [!IMPORTANT]
+> We highly recommend creating a backup before enabling the Swarming feature.
+
+## Prerequisites
+
+- DataMiner 10.5.1 or higher.
+- [STaaS](xref:STaaS) or a [dedicated clustered storage](xref:Configuring_dedicated_clustered_storage) setup
+- No [Failover Agents](xref:About_DMA_Failover) are present in the cluster.
+- No [offload database](xref:Offload_database) is configured.
+- Enhanced services in the cluster (if any) use only [compatible connectors](xref:SwarmingPrepare).
+- Scripts (Automation/GQI) and QActions in connectors have all been [made compatible](xref:SwarmingPrepare), i.e. they do not use obsolete or incompatible SLNet calls/properties that handle Alarm IDs as "DmaID/AlarmID".
+- For the user account that will be used to enable Swarming, the [Admin Tools](xref:DataMiner_user_permissions#modules--system-configuration--tools--admin-tools) user permission is enabled.
+
+## Running a prerequisites check
+
+To verify whether your system meets these prerequisites before you enable Swarming, you can send a *SwarmingPrerequisitesCheckRequest* message to SLNet using the [SLNetClientTest tool](xref:SLNetClientTest_tool). The prerequisite check is also executed when you enable Swarming, ensuring that all prerequisites are met before Swarming is enabled.
+
+> [!NOTE]
+> The prerequisites check can take some time, as checking the usage of legacy alarm IDs in scripts can take a while.
```

Comment:
should we give an indication?
eg "multiple minutes" 

"Some time" is a bit ambiguous and 20s in the context of slnet msg is also already "some time"...